Lesson2/js/timer.js

```diff
@@ -0,0 +1,69 @@
+import { Howl } from 'howler'
+import { printError, deleteError } from './output.js';
+import soundFile from '../audio/sound.mp3'
+let userInput = document.getElementById('input');
+let seconds = document.getElementById('second');
+let minutes = document.getElementById('minute');
+let totalSeconds = 0;
+let countdown = null;
+
+
+const sound = new Howl({
+    src: ['./audio/sound.mp3'],
```

Comment:
Путь до файлика в сборке уже лежит в soundFile)
```suggestion
    src: [soundFile],
```